src/components/AddHogForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react"
+
+function AddHogForm({updateHogList}) {
+  const [hogName, setHogName] = useState("")
+  const [hogSpecialty, setHogSpecialty,] = useState("")
+  const [hogWeight, setHogWeight,] = useState("")
+  const [hogImage, setHogImage,] = useState("")
+  const [hogGreased, setGreased,] = useState(true)
+  const [hogMedals, setMedals] = useState("")
+
+  const handleHogName = event => setHogName(event.target.value)
+
+  const handleHogSpecialty = event => setHogSpecialty(event.target.value)
+
+  const handleHogWeight = event => setHogWeight(event.target.value)
+
+  const handleHogImage = event => setHogImage(event.target.value)
+
+  const handleHogMedals = event => setMedals(event.target.value)
```

Comment:
Review the form abstraction lesson to see how you could simply this substantially.

Reply:
I see, if I have time to go back I will and try to implement it. I will implement like this but with my form values. 

` const [formData, setFormData] = useState({
    firstName: "John",
    lastName: "Henry",
  });`

Reply:
Yes, exactly. There will be more opportunities to practice, so don't worry about going back for now.